research/src/pages/popup/popup.research.explainer.mdx

```diff
@@ -83,59 +93,97 @@ So this markup represents popup content:
+As mentioned above, a `<div popup=popup>` will be hidden by default. If it is desired that the popup should be shown automatically upon page load, the `defaultopen` attribute can be applied:
 
 ```html
-<button triggerpopup=mypopup>Click me</button>
-<div id=mypopup popup=popup>Popup content</div>
+<div popup=popup defaultopen>
 ```
 
-When the button in this example is activated, the UA will call `.showPopup()` on the `<div id=mypopup>` element. In this way, no Javascript will be necessary for this use case.
+In this case, the UA will immediately call `showPopup()` on the element, as it is parsed. If multiple such elements exist on the page, only the first such element (in DOM order) on the page will be shown.
 
-When the `triggerpopup` attribute is applied to an activating element, the UA may automatically map this attribute to an appropriate `aria-*` attribute, such as `aria-haspopup`, `aria-describedby` or `aria-expanded`. There will need to be further discussion with the ARIA working group to determine the exact ARIA semantics, if any, are necessary.
+Note that `hint` popups cannot use the `defaultopen` attribute:
 
-## Page Load Trigger (the `initiallyopen` attribute)
+```html
+<div popup=hint defaultopen>I will not be shown on page load</div>
+```
 
-As mentioned above, a `<div popup=popup>` will be hidden by default. If the popup should instead be shown upon page load, the `initallyopen` attribute can be applied:
+Note also that more than one `async` popup can use `defaultopen` and all such popups will be shown on load, not just the first one:
 
 ```html
-<div popup=popup initiallyopen>
+<div popup=async defaultopen>Shown on page load</div>
+<div popup=async defaultopen>Also shown on page load</div>
 ```
 
-In this case, the UA will immediately call `showPopup()` on the element, as it is parsed. If multiple such elements exist on the page, only the first such element (in DOM order) will be shown.
 
+### Shown vs Hidden Popups
+
+The styling for a popup is provided by roughly the following UA stylesheet rules:
+
+```css
+[popup i]:not(:popup-open) {
+  display: none;
+}
+
+[popup i] {
+  position: fixed;
+}
+```
+
+The above rules mean that a popup, when not "shown", has `display:none` applied, and that style is removed when one of the methods above is used to show the popup. Note that the `display:none` UA stylesheet rule is **not** `!important`. In other words, developer style rules can be used to override this UA style to make a not-showing popup visibile in the page. In this case, the popup will **not** be displayed in the top layer, but rather at it's ordinary z-index position within the document. This can be used, for example, to animate the show/hide behavior of the popup, or make popup content "return to the page" instead of becoming hidden.
```

Comment:
```suggestion
The above rules mean that a popup, when not "shown", has `display:none` applied, and that style is removed when one of the methods above is used to show the popup. Note that the `display:none` UA stylesheet rule is **not** `!important`. In other words, developer style rules can be used to override this UA style to make a not-showing popup visible in the page. In this case, the popup will **not** be displayed in the top layer, but rather at it's ordinary `z-index` position within the document. This can be used, for example, to animate the show/hide behavior of the popup, or make popup content "return to the page" instead of becoming hidden.
```

Reply:
Thanks! done.